src/actions/tests/pause.spec.js

```diff
@@ -2,7 +2,8 @@ import {
   actions,
   selectors,
   createStore,
-  getHistory
+  getHistory,
+  makeSource
```

Comment:
Are these calls fixes for errors that had been previously being ignored? Usually one should submit unrelated fixes as a separate PR, and treat the harness fixes as blocked on that.

Reply:
yep, this is a tad lazy... 

I don't think we gain anything in version control land by doing that though here